src/needless_continue.rs

```diff
@@ -0,0 +1,547 @@
+//! Checks for continue statements in loops that are redundant.
```

Comment:
This comment should be right before the `declare_lint!` invocation. Also it should have the usual form – look at the other lints for example.